packages/gatsby-theme-parliament/src/components/Layout/index.js

```diff
@@ -164,7 +164,6 @@ export default ({ children, pageContext, location }) => {
             gridArea="sidenav"
             isHidden={!hasSideNav}
             position="fixed"
-            overflow="auto"
```

Comment:
If we have overflow set to auto the popover for the search results gets cut off as it will go under the main content. I'd say if sidenav items overflow the container it shouldn't cause a scroll bar to appear, rather they should wrap to another line. This is more of a question for LIse though.